src/plugins/data/server/data_views/saved_objects_client_wrapper.test.ts

```diff
@@ -12,7 +12,7 @@ import { SavedObjectsClientContract } from 'src/core/server';
 import { DataViewSavedObjectConflictError } from '../../common/data_views';
 
 describe('SavedObjectsClientPublicToCommon', () => {
-  const soClient = ({ resolve: jest.fn() } as unknown) as SavedObjectsClientContract;
```

Comment:
The linter was complaining about this 🤷‍♂️

Reply:
This line I owned by @elastic/kibana-app-services 🤦‍♂️

Reply:
I think you can drop this change after https://github.com/elastic/kibana/commit/c76082e00699a651d1494978fc3df994be1b59b0 was merged